ui.frontend/src/main/webpack/site/js/components/buildFacet.ts

```diff
@@ -49,7 +47,7 @@ BuildFacetOption): HTMLDivElement => {
     : false
 
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
-  facetInput.addEventListener('change', async (event) => {
+  facetInput.addEventListener('click', async (event) => {
```

Comment:
@ana-zdravkovikj that was 'change' not 'click' :D

Reply:
Done